src/Microsoft.OpenApi/Validations/OpenApiValidator.cs

```diff
@@ -170,7 +169,15 @@ private void Validate<T>(T item)
         private void Validate(object item, Type type)
         {
             if (item == null) return;  // Required fields should be checked by higher level objects
-            var rules = _ruleSet.Where(r => r.ElementType == type);
+
```

Comment:
nit, per style guideline and consistency, please convert this to 

if ( item == null ) 
{
    return;
}